irohad/main/irohad.cpp

```diff
@@ -96,6 +96,14 @@ DEFINE_validator(verbosity, validateVerbosity);
 std::promise<void> exit_requested;
 
 int main(int argc, char *argv[]) {
+  if (std::string(IROHA_COMMIT_HASH) == "") {
+    gflags::SetVersionString("Unknown");
+  } else {
+    std::stringstream version;
```

Comment:
Maybe put it into the log instead? Otherwise, submitters can lose version in bug reports.